app/src/main/java/com/example/trainingintive/what_to_do_feature/data/mapper/ApiToDomain.kt

```diff
@@ -0,0 +1,5 @@
+package com.example.trainingintive.what_to_do_feature.data.mapper
+
+import com.example.trainingintive.what_to_do_feature.data.entity.ActivityModelFromApi
+
+fun ActivityModelFromApi.toDomain() = com.example.trainingintive.what_to_do_feature.domain.model.ActivityModel(activity)
```

Comment:
1. New line after "="
2. Why you are using full name (with package)?
3. You can keep mappers in files with the class declaration, this way it will be simpler to find it, otherwise, you will be left with a loooooot of files with only one mapper